framework/doc/content/source/problems/FEProblemBase.md

```diff
@@ -0,0 +1,41 @@
+# FEProblemBase
+
+The FEProblemBase class is an intermediate base class containing all of the common
+logic for running the various MOOSE simulations. MOOSE has two built-in types of
+problems [FEProblem.md] for solving "normal" physics problems and [EigenProblem.md]
+for solving Eigenvalue problems. Additionally, MOOSE contains an [ExternalProblem.md]
+problem useful for creating ("MOOSE-wrapped Apps")[moose_wrapped_apps.md].
```

Comment:
```suggestion
problem useful for creating ["MOOSE-wrapped Apps"](moose_wrapped_apps.md).
```

Reply:
The above suggestion fixes the syntax; however, this will still fail to build because the moose_wrapped_apps.md is in the modules/doc, not the framework and the framework pages are required to build stand-alone.

There are three solutions:

1. Remove this link
2. Mark it as optional: `[foo](foo.md optional=True)`
3. Move the linked page into framework/doc